assets/src/components/d3/createHistogram.js

```diff
@@ -55,7 +55,7 @@ function createHistogram ({ data, width, height, domElement, xAxisLabel, yAxisLa
   main.append('div')
     .attr('aria-live', 'polite')
     .attr('id', 'grade-view-narrative')
-    .attr('class', 'screenreader-only sr-only')
+    .attr('class', 'fa-sr-only')
```

Comment:
I was confused at first when playing around with `fontawesome` approaches why this was being affected, so I changed it to just use the `fa` prefix for clarity. I didn't see `screenreader-only` applying any styles.

Reply:
Can you point me to the documentation for `fa-sr-only`?

Reply:
Thanks! When I added accessibility support for d3 visuals, I added those based on Gonzalo's recommendation. I don't remember too many details now. I felt it is best I will test it now.

Reply:
I don't think we are using `fontawesome` fonts for histogram. We only rely on the d3/svg fonts. why was this change needed? Especially this line is making sure that screen reader can read the text for accessibility. 

I don't see any harm done when I turn on voice over things are working fine. But Trying to understand change here.

Reply:
Well, when I removed `fontawesome` during testing, I noticed that the screenreader text you added was being shown visually. So after analyzing the styles, I realized `fontawesome` classes were being used to make the text screenreader only. I think `sr-only` must have been an alias or alternative for `fa-sr-only`; I changed it to `fa-sr-only` because I wanted us to be able to tell later that it came from `fontawesome` (`fa-`). I don't think the `screenreader-only` class was necessary or adding any styles, so I removed it.

Does that help explain?